geoportailv3/static/js/olcs/lux3dmanager.js

```diff
@@ -24,10 +24,27 @@ app.olcs.Lux3DManager = class extends ngeo.olcs.Manager {
      */
     this.ngeoLocation_ = ngeoLocation;
 
+    /*
+     * A factor used to increase the screen space error of terrain tiles when they are partially in fog. The effect is to reduce
+     * the number of terrain tiles requested for rendering. If set to zero, the feature will be disabled. If the value is increased
+     * for mountainous regions, less tiles will need to be requested, but the terrain meshes near the horizon may be a noticeably
+     * lower resolution. If the value is increased in a relatively flat area, there will be little noticeable change on the horizon.
+     * type {Number}
+     * default 2.0
```

Comment:
`25` seems to be the default in [ol-cesium](https://github.com/openlayers/ol-cesium/blob/master/src/olcs/contrib/manager.js#L81)